pennylane/fourier/__init__.py

```diff
@@ -15,15 +15,16 @@
 of quantum circuits."""
 import warnings
 from .coefficients import coefficients
-from .simple_spectrum import simple_spectrum
-from .advanced_spectrum import advanced_spectrum
+from .circuit_spectrum import circuit_spectrum
+from .qnode_spectrum import qnode_spectrum
+from .utils import join_spectra, get_spectrum
 
 
 def spectrum(*args, **kwargs):
-    """Alias for ``simple_spectrum``. To be removed soon."""
+    """Alias for ``circuit_spectrum``. To be removed soon."""
```

Comment:
```suggestion
    """Alias for :func:`~.circuit_spectrum`. To be removed soon."""
```

Since this appears in the docs :) 

![image](https://user-images.githubusercontent.com/2959003/136150943-e837ff20-4963-4486-b623-14a9e50777b2.png)